lib/game/api/GameApi.jsx

```diff
@@ -0,0 +1,104 @@
+import CONSTANTS from "../../../const";
```

Comment:
That's better. Why placing this file in such a deep directory? You're planning to have so many thing in `lib`?

Reply:
Sorry it wasn't clear, I was referring to the GameApi file.